contrib/devtools/fix-copyright-headers.py

```diff
@@ -16,7 +16,7 @@
 import re
 
 year = time.gmtime()[0]
-CMD_GIT_DATE = 'git log --format=@%%at -1 %s | date +"%%Y" -u -f -'
+CMD_GIT_DATE = 'git log --format=%%ad --date=short -1 %s | cut -d"-" -f 1'
```

Comment:
Lazy<sup>TM</sup> and ItWorks<sup>TM</sup>, also you need to call at least `git` to make this script work. Though, I agree that calling perl is not required. (I won't change it in this pull, though.)